diesel/src/pg/connection/result.rs

```diff
@@ -69,7 +71,7 @@ impl PgResult {
         PgRow::new(self, idx)
     }
 
-    pub fn get(&self, row_idx: usize, col_idx: usize) -> Option<&[u8]> {
+    pub fn get(&self, row_idx: usize, col_idx: usize) -> Option<PgValue> {
```

Comment:
We need to load the oid for each type here somehow.
Postgresql provided `PQftype(PqResult*, usize*)` for this.
I'm not sure if that does a additional client->server round trip or just reads the value from same internal libpq field. In the second case we can just call `PQftype` here, otherwise this should be done while constructing `PQResult` above.
cc @sgrif because I'm not sure how libpq works internally

Reply:
I'll test the ftype call and see if I get valid OIDs out of it!